go/vt/sqlparser/Makefile

```diff
@@ -5,7 +5,7 @@
 MAKEFLAGS = -s
 
 sql.go: sql.y
-	go tool yacc -o sql.go sql.y
+	goyacc -o sql.go sql.y
```

Comment:
I don't seem to have `goyacc`, but `go tool yacc` works for me. Am I missing something, or was there a reason for this change?